### PR TITLE
Restart apache by removing pid - closes #75

### LINKED ISSUE #75: Cannot restart apache container
The entrypoint should remove the pid

### PR DESCRIPTION
**Pull request by anaelChardan**

### CODE REVIEW

File: apache-php/Dockerfile [hunk 1]
```diff
@@ -27,8 +27,11 @@ RUN sed -i "s/export APACHE_RUN_USER=www-data/export APACHE_RUN_USER=docker/" /e
...
 
+COPY files/apache-foreground /usr/local/bin
+RUN  chmod +x /usr/local/bin/apache-foreground
+
 # Define "docker" as current user
 USER docker
 
 # Run apache in foreground
-CMD ["sudo", "/usr/sbin/apache2ctl", "-D", "FOREGROUND"]
+CMD ["sudo", "apache-foreground"]
```
Comment (damien-carcel) [inline apache-php/Dockerfile]: /usr/local/bin/apache-foreground

File: apache-php/files/apache-foreground [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+rm -f /var/run/apache2/apache2.pid
```
Comment (damien-carcel) [inline apache-php/files/apache-foreground:4]: 👍

File: apache-php/files/apache-foreground [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+exec /usr/sbin/apache2ctl -DFOREGROUND
```
Comment (damien-carcel) [inline apache-php/files/apache-foreground]: Please add a space between `-D` and `FOREGROUND`

**Review by damien-carcel — CHANGES_REQUESTED** [review]

File: apache-php/Dockerfile (hunk 1 above)
Comment (anaelChardan) [reply apache-php/Dockerfile]: It work without this but I'll correct.

File: apache-php/files/apache-foreground (hunk 3 above)
Comment (anaelChardan) [reply apache-php/files/apache-foreground]: Almost all images does not put a space between them, I didn't know :D, I'll correct

**Comment by anaelChardan** [comment]
Corrected

File: apache-php/Dockerfile (hunk 1 above)
Comment (damien-carcel) [reply apache-php/Dockerfile]: Thanks. I just prefer to always put the full path for commands (even if there probably some files where it is not the case, and it should be).

**Review by damien-carcel — APPROVED** [review]

**Pull request merged**